bin/et

```diff
@@ -9,25 +9,19 @@ set -e
 # Needed because if it is set, cd may print the path it changed to.
 unset CDPATH
 
-# On Mac OS, readlink -f doesn't work, so follow_links traverses the path one
-# link at a time, and then cds into the link destination and find out where it
-# ends up.
-#
-# The function is enclosed in a subshell to avoid changing the working directory
-# of the caller.
-function follow_links() (
```

Comment:
This code is copy-pasted all over the engine repo. If this change is a better way to do things, should we change all occurrences?

Reply:
Interesting -- I'll set a reminder for a week and if no one's raised any issues, it seems like a cleanup worth doing.